frontend/src/scenes/trends/PersonsModal.tsx

```diff
@@ -94,10 +94,7 @@ export function PersonsModal({
     )
 
     const isDownloadCsvAvailable: boolean = view === InsightType.TRENDS && showModalActions && !!people?.action
-    const isSaveAsCohortAvailable =
-        clickhouseFeaturesEnabled &&
-        (view === InsightType.TRENDS || view === InsightType.STICKINESS || view === InsightType.FUNNELS) &&
-        showModalActions
+    const isSaveAsCohortAvailable = clickhouseFeaturesEnabled && showModalActions
```

Comment:
what about lifecycle and retention? 👀

Reply:
Good call. Retention uses a different modal so this wont impact, but forgot about lifecycle. Also, those still need a tiny bit of API additions. I just readded the conditions